ssl/ssl_lib.c

```diff
@@ -1716,6 +1716,10 @@ int SSL_shutdown(SSL *s)
 
 int SSL_renegotiate(SSL *s)
 {
+    /* Do nothing in TLS1.3 */
```

Comment:
Ok, I changed it to return an error, and added a TODO to revist this when we have KeyUpdate support.